src/WP_CLI/CommandWithTerms.php

```diff
@@ -121,11 +121,12 @@ public function list_( $args, $assoc_args ) {
 	 * : The name of the term's taxonomy.
```

Comment:
Should we update this one too? Are there others in the file?

Reply:
I don't think so. By default, the slug is the taxonomy key but it can be different. I've updated the doc of the command `set` that was using `slug` instead of `name` to the `taxonomy` option.